src/components/DatatableV2/tests/selection.test.tsx

```diff
@@ -392,4 +392,38 @@ describe('DatatableV2/selection', () => {
       expect(screen.getAllByLabelText('Toggle select row')[1]).toBeChecked();
     });
   });
+
+  describe('with "selectAllMode=virtual"', () => {
+    it('should display the "select all" prompt when all rows are selected', async () => {
```

Comment:
@mrthebunny Could you please add case when user selects all row and then deselect one row